Source/dom/models/Selection.js

```diff
@@ -37,7 +37,7 @@ export class Selection {
     Object.defineProperty(this, 'length', {
       enumerable: true,
       get() {
-        return this._object.selectedLayers().layers().count()
+        return this._object.selectedLayers().count()
```

Comment:
As I understand it, it's B cc @bomberstudios

Reply:
I can confirm it’s B. You can tell because that line uses `_object`, which basically means “drop down to internal objects, this should not be used publicly”